tracing-subscriber/src/fmt/mod.rs

```diff
@@ -57,7 +57,7 @@
 //!     .finish();
 //! ```
 //!
-//! You can find the configuration methods for [`FmtSubscriber`] in [`fmt::Builder`].
+//! You can find the configuration methods for [`FmtSubscriber`] in [`fmt::SubscriberBuilder`].
 //!
```

Comment:
it could be good if the mod-level docs also mentioned the layer and talked about how it should be used...we can add that in a follow-up though...